packages/nc-gui/components/account/UserList.vue

```diff
@@ -182,10 +182,10 @@ const openDeleteModal = (user: UserType) => {
           <GeneralLoader size="xlarge" />
         </div>
         <!-- if users are empty -->
-        <div v-else-if="!users.length" class="flex items-center justify-center text-center h-128.25">
+        <div v-else-if="!users.length" class="flex items-center justify-center text-center h-full">
           <a-empty :image="Empty.PRESENTED_IMAGE_SIMPLE" :description="$t('labels.noData')" />
         </div>
-        <section v-else class="tbody">
+        <section v-else class="tbody h-[calc(100%-3.3rem)] nc-scrollbar-md border-t-0">
```

Comment:
Add `!overflow-auto` here is empty scrollbar will not look good

Reply:
And use `h-[calc(100%-3rem)]` instead of `h-[calc(100%-3.3rem)]` as it 's casing dead space below when we have scroll

<img width="986" alt="image" src="https://github.com/nocodb/nocodb/assets/35809690/dc7921a5-39df-482c-a88a-b6925f451970">